version.php

```diff
@@ -24,8 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2019062601;    // The current plugin version (Date: YYYYMMDDXX).
-$plugin->release   = 2019062601;    // Match release exactly to version.
+$plugin->version   = 2020031800;    // The current plugin version (Date: YYYYMMDDXX).
+$plugin->release   = 2020031800;    // Match release exactly to version.
 $plugin->requires  = 2014051200;    // Requires this Moodle version. (2.7)
 $plugin->component = 'auth_saml2';  // Full name of the plugin (used for diagnostics).
-$plugin->maturity  = MATURITY_STABLE;
+$plugin->maturity  = MATURITY_STABLE;
```

Comment:
all looks great except this tiny whitespace EOL

Reply:
Hi @brendanheywood, thank you. Done!